### PR TITLE
Added awesome-mac to the awesome list

### PR DESCRIPTION
**Pull request by xyNNN**
Hey folks,

awesome-mac is a curated list of amazingly awesome Mac OS X tools, applications and games.

### CODE REVIEW

**Comment by arthurvr** [comment]
https://github.com/sindresorhus/awesome/blob/master/contributing.md

**Comment by arthurvr** [comment]
Also, the issues on the repo aren't enabled.

**Comment by xyNNN** [comment]
Activated the issue feature on this repository. What else is missing?!

**Comment by arthurvr** [comment]
> Link additions should be added to the bottom of the relevant category.

**Comment by xyNNN** [comment]
Hm, I don't know what you mean with that?! Can you give me an example? :)

**Comment by arthurvr** [comment]
Just move the link to the bottom of the category :)

File: readme.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ If you want to create your own list, please read the [create list instructions](
 
 - [Node.js](https://github.com/sindresorhus/awesome-nodejs)
 - [Frontend Development](https://github.com/dypsilon/frontend-dev-bookmarks)
+- [Mac OS X](https://github.com/xyNNN/awesome-mac)
```
Comment (sindresorhus) [inline readme.md]: It's named `OS X` now, not `Mac OS X`.

**Comment by sindresorhus** [comment]
Gonna pass on this for now. There's very little content there now. Happy to reconsider some time in the future.

The license used is also not suitable for text content: https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license

**Pull request closed without merging**